app/seed.js

```diff
@@ -0,0 +1,57 @@
+const Article = require('./models/article')
+const User = require('./models/user')
+const Counter = require('./models/routeCounter')
+const logger = require('./logger')
+const {articles, users} = require('./staticdata/dummydata.json')
+const entities = require('he')
+const SPACES = /\s/g, DASH = '-'
+const DUMMY_PASSWORD = 'mattlovestrees23'
+const {removeOrphanedImages} = require('./utils')
+
+async function initialLaunchCheck(options) {
+    if(!process.env.DEV_MODE) return
+    if(options?.clear_db && process.env.DEV_MODE) await dropCollections()
+    try {
+        const articleCount = await Article.estimatedDocumentCount()
+        const userCount = await User.estimatedDocumentCount()
+        if(userCount === 0) await seedUsers()
+        if(articleCount === 0) await seedArticles()
+
+    } catch(err) {
+        logger.info(`SeedDB Error: ${err}`)
+    }
+}
+
+async function seedArticles() {
```

Comment:
Please seed also articles view count to 0 for each article

Reply:
Done